Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -573,6 +609,38 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         webviewCloseButton.tap()
     }
 
+    func testBLIKPaymentMethod() throws {
```

Comment:
Do we need this if we have `testBLIKPaymentMethodPolling` what is it testing that `testBLIKPaymentMethodPolling` isn't?